extensions/narayana-jta/deployment/src/test/java/io/quarkus/narayana/interceptor/TransactionalTest.java

```diff
@@ -16,15 +20,20 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
+import io.quarkus.bootstrap.classloading.ClassLoaderLimiter;
 import io.quarkus.test.QuarkusUnitTest;
 
 public class TransactionalTest {
 
+    static final Set<String> loadedClasses = Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
I guess this was later replaced with your nice new testing helper?

It's not used.

Reply:
oops